pkg/logs/input/kubernetes/launcher.go

```diff
@@ -186,14 +186,26 @@ func (l *Launcher) getSource(pod *kubelet.Pod, container kubelet.ContainerStatus
 	}
 	cfg.Type = config.FileType
 	cfg.Path = l.getPath(basePath, pod, container)
-	cfg.Identifier = container.ID
+	cfg.Identifier = getTaggerEntityID(container.ID)
```

Comment:
you should call  `kubelet.KubeContainerIDToTaggerEntityID(...)`  and manage the `error` case here.

Reply:
done